inference-engine/ie_bridges/c/include/c_api/ie_c_api.h

```diff
@@ -274,6 +274,7 @@ typedef struct input_shapes {
 }input_shapes_t;
 
 /**
+ * @deprecated Use new ie_blob_map_t structure instead
```

Comment:
we also need to introduce a macro like `INFERENCE_ENGINE_C_DEPRECATED` (see implementation here https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/include/ie_api.h#L38)

Reply:
and mark with such macro all deprecated symbols